src/pages/research_profile/ResearchProfile.tsx

```diff
@@ -53,57 +39,32 @@ const ResearchProfile = () => {
     ],
   });
 
-  const fullName = person?.names ? getFullCristinName(person.names) : '';
-  const orcidUri = getOrcidUri(person?.identifiers);
-  const activeAffiliations = person?.affiliations ? filterActiveAffiliations(person.affiliations) : [];
-
   return (
-    <StyledPageContent>
-      <PageHeader>{fullName}</PageHeader>
-      {isLoadingPerson || isLoadingRegistrations ? (
+    <Box sx={{ display: 'grid', gridTemplateColumns: '3fr 1fr', gap: '1rem' }}>
+      {isLoadingRegistrations ? (
```

Comment:
Nå vil alltid person hentes både her og i `ResearchProfilePanel` som gir litt ekstra overhead. Lurer på om det kan være en ide å beholde sjekken her, og sende inn `person` som prop til `ResearchProfilePanel` slik at vi ikke trenger å hente igjen om den allerede er der?
```suggestion
      {isLoadingRegistrations || isLoadingPerson ? (
```